src/einsteinpy/plotting/geodesics_static.py

```diff
@@ -1,4 +1,6 @@
+import math
```

Comment:
It is better to stay in numpy ecosystem :)

Reply:
Yeah, Done :)